packages/taquito-rpc/src/types.ts

```diff
@@ -182,7 +182,7 @@ export interface InlinedPreEndorsement {
   signature?: string;
 }
 
-export type OperationContentsBallotEnum = 'nay' | 'yay' | 'pass';
+export type BallotEnum = 'nay' | 'yay' | 'pass';
```

Comment:
nit should we rename since its not an actual enum?

Reply:
it's still an enum just without an initializer

Reply:
sure, do you have a name in mind?

Reply:
the three that come to mind are BallotVote, VotingOptions, BallotVotingOptions just a little more descriptive

Reply:
I like BallotVote as well, I'll update it